src/AssetBundles/GulpBundleAssetsBundles.php

```diff
@@ -36,7 +36,7 @@ abstract class GulpBundleAssetsBundles implements AssetBundlesInterface
     public function __construct($filePath)
     {
         if (!is_string($filePath)) {
-            throw new \InvalidArgumentException('$filePath must of type string but was ' . gettype($filePath));
+            throw new \InvalidArgumentException('$filePath must of type string but was ' . gettype($filePath));// @codeCoverageIgnore
```

Comment:
I've learned something new today.

Could be worth having a test anyway for deprecated code, tests that could be removed later.

Reply:
Annoying thing about the deprecated code is that nothing on our side was using it. In most instances, we don't have a way to test without adding technical debt. Here though, its just a typehint polyfill. I'm not going to bother to test something so trivial, extremely easy to get into a hole and lose time on something that will never offer real value.